app/src/main/res/values/strings.xml

```diff
@@ -15,6 +15,7 @@
     <string name="week_at_a_glance_meal_deleted">Removed meal</string>
     <string name="week_at_a_glance_undo_delete">Undo</string>
     <string name="week_at_a_glance_delete_menu">Delete</string>
+    <string name="week_at_a_glance_edit_servings_menu">Edit Servings</string>
```

Comment:
Maybe add ellipses 'cause you're opening a dialog? (If you add this, use `&#8230;` instead of `...`, because it looks prettier)